### ISSUE TITLE
What's the minimal rust version?

### ISSUE THREAD

**Issue by Endle** [sunng87/cargo-release#286]
I'm using rust 1.49.0  

When installing this package, I hit E0658
```
$ cargo install cargo-release

error[E0658]: use of unstable library feature 'str_split_once': newly added
   --> /home/lizhenbo/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-4f7dbcce21e258a2/cargo-release-0.16.3/src/version.rs:113:63
    |
113 |             if let Some((alpha, numeric)) = self.pre.as_str().split_once(".") {
    |                                                               ^^^^^^^^^^
    |
    = note: see issue #74773 <https://github.com/rust-lang/rust/issues/74773> for more information

```

What's the minimal rust version for cargo-release?

Thank you

**Comment by epage** [comment]
1.52.

I recommend checking [CI](https://github.com/sunng87/cargo-release/blob/master/.github/workflows/rust.yml#L35) as the definitive in this. Even if we document it somewhere, it is likely to be missed as we update CI.

Granted, that can be confusing because we have a stale travis file that we aren't using.

**Comment by sunng87** [comment]
Let me remove that travis config to avoid confusion.

**Comment by sunng87** [comment]
Fixed in 6b731bb
